staff/pere-hernandez/bash/README.md

```diff
@@ -0,0 +1,141 @@
+# BASH
+
+Command-line interface comands typical in BASH
+
+## pwd --- Shows the current directory ---
+
+```sh
+$ pwd
+```
+
+## ls --- Shows the files inside the current directory---
+
+```sh
+$ ls
+```
+
+## ls -l --- Show's the current directory's permits---
+
+```sh
+$ ls -l
+```
+
+## touch hola.js--- Creates a new file called 'hola.js'---
+
+```sh
+$ touch
```

Comment:
The name of the file you are creating should appear next to the "touch" command 🧐

Reply:
Done